tests/bluetooth/mesh/brg/testcase.yaml

```diff
@@ -0,0 +1,10 @@
+tests:
+  bluetooth.mesh.brg:
+    platform_allow:
+      - native_posix
```

Comment:
native_posix is deprecated. Better don't add new tests running on it does not provide any extra coverage for the test itself. (otherwise we will need to remove this lines very soon when we remove the platform)

Reply:
Removed here: https://github.com/zephyrproject-rtos/zephyr/pull/78400